bundler/lib/bundler/cli/outdated.rb

```diff
@@ -31,9 +31,9 @@ def run
       end
 
       Bundler.definition.validate_runtime!
-      current_specs = Bundler.ui.silence { Bundler.definition.resolve }
+      current_specs = Bundler.ui.with_output_stream(:stderr) { Bundler.definition.resolve }
 
-      current_dependencies = Bundler.ui.silence do
+      current_dependencies = Bundler.ui.with_output_stream(:stderr) do
```

Comment:
On second thought, I'll revert these changes, since they don't have anything to do with the `--parseable` option. The only edit needed in this file is line 57, below.